@here/harp-webtile-datasource/test/WebTileTest.ts

```diff
@@ -22,6 +23,20 @@ describe("WebTileDataSource", function() {
         projection: mercatorProjection
     } as MapView;
 
+    let loggerWasEnabled = true;
```

Comment:
this is quite a bit of code to remove the error logging, can it be done in a different way?

Also, it seems that at least in the WebTileTest.ts there is only a single test which causes this. So I think it makes sense to put this code just there.

Reply:
Moved to the test in question. I don't know how to simplify it, let me know if you think of something. 

We have to leave the logging system in its original state after the affected state to avoid affecting other tests. We have some tests that even capture logs and make assertions on them (quite nasty if you ask me).